apps/weshipit/components/global-seo.tsx

```diff
@@ -0,0 +1,31 @@
+import { DefaultSeo } from 'next-seo';
+
+interface GlobalSeoProps {
+  data: {
+    seo_description?: string;
+    seo_title?: string;
+  };
+}
+
+export default function GlobalSeo({ data }: GlobalSeoProps) {
+  const seoTitle = data.seo_title;
+  const seoDescription = data.seo_title;
+  return (
+    <>
+      <DefaultSeo
```

Comment:
do you think you can merge the content from https://github.com/flexbox/weshipit.today/blob/49f934cab9b97ce82f3a56089ffb0b5aa5db4d33/apps/weshipit/components/next-head.tsx